secp256k1.nim

```diff
@@ -517,3 +517,7 @@ proc default*(T: type SkSignature): T {.error: "loophole".}
 proc default*(T: type SkRecoverableSignature): T {.error: "loophole".}
 proc default*(T: type SkEcdhSecret): T {.error: "loophole".}
 proc default*(T: type SkEcdhRawSecret): T {.error: "loophole".}
+
+func privKeyTweakAdd*(secretKey: var openArray[byte], tweak: openArray[byte]) =
```

Comment:
```suggestion
func tweakAdd*(secretKey: var SkSecretKey, tweak: openArray[byte]) =
```

also, can use the `secp256k1_context_no_precomp` context in this case - when in doubt, check the rust secp wrapper.

finally, might as well expose the other tweak functions (pubkey, mul) as well if we're exporting this one